lib/pirate.rb

```diff
@@ -11,4 +11,7 @@ def initialize(name, job = 'Scallywag', act = 0, booty = 0)
     @booty = booty
   end
 
+  def ship_robbery
```

Comment:
@Taamir-Ransome better name would be `rob_ship` or maybe even `rob_ship!` see if you can find what it means when a method in ruby ends with a `!` (bang)